### DISCUSSION TITLE
jsonSerialize(): array

### DISCUSSION THREAD

**Discussion by arokito** [spatie/laravel-medialibrary#2867] [Q&A]
Declaration of Spatie\MediaLibrary\MediaCollections\Models\Collections\MediaCollection::jsonSerialize() must be compatible with Illuminate\Support\Collection::jsonSerialize(): array

**Comment by PaolaRuby** [comment]
Upgrade your package

**Reply by itskawsar** [reply]
> Upgrade your package

still showing the error :(

Version testing:
spatie/laravel-medialibrary v10.0.0
PHP v8.0.11
Lavavel v9.7.0
